src/main/steps/c100-rebuild/childaddress/content.ts

```diff
@@ -0,0 +1,78 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn, isInvalidPostcode } from '../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  headingTitle: 'Where do the children live?',
+  paragraph1: 'Please tell us the postcode of the children you’re making this application about.',
+  paragraph2: `This information will be used to identify which court will handle your application.
+                If the children have a different postcode, enter the one that is most 
+                convenient to most children in the application.`,
+  warningText: `You should only enter your own postcode if the children live with you at the address,
+                 or you don't know where the children are living.`,
+  postcodeLabel: 'Postcode',
+  detailsLabel: "Why we use the term 'children'",
+  detailsContent: `We use ‘children’ as a general term to mean whether you have a child or children.
+                   We do this to avoid repetition.`,
+  errors: {
+    postcode: {
+      required: 'Enter a valid full postcode, with or without a space',
+      invalid: 'Enter a valid full postcode, with or without a space',
+    },
+  },
+});
+
+const cy = () => ({
+  headingTitle: 'Where do the children live? - welsh',
+  paragraph1: `Please tell us the postcode of the children 
+              you’re making this application about. - welsh`,
+  paragraph2: `This information will be used to identify which court will handle your application.
+                If the children have a different postcode, enter the one that is most 
+                convenient to most children in the application. - welsh`,
+  warningText: `You should only enter your own postcode if the children live with you at the address,
```

Comment:
We can define warning text like below
`warningText: {
    text: 'The information that you give in this section will also be shared with the other people in this application, so they are able to respond to what you have said.',
    iconFallbackText: 'Warning',
  },`

Reply:
Changes done